worker.go

```diff
@@ -211,3 +241,12 @@ func go_frankenphp_finish_request(mrh, rh C.uintptr_t, deleteHandle bool) {
 		c.Write(fields...)
 	}
 }
+
+func resetOpCache() {
+	success := C.frankenphp_execute_php_function(C.CString("opcache_reset"))
```

Comment:
The Cstring doesn't look freed at anytime.

Reply:
👍 Is fixed, though cgo frees them itself.